css/style.css

```diff
@@ -0,0 +1,562 @@
+  border-bottom: 1px solid var(--border);
+  padding-right: 50px;
+  user-select: none;
+  cursor: pointer;
+}
+
+.expanded > .about-myself_menu-item {
+  border: none;
+}
+
+.expanded > .about-myself_menu-item > img {
+  rotate: 90deg;
+}
+
+.contact-form {
+  background-color: var(--white);
+}
+
+.contact-form__wrapper {
+  display: flex;
+  padding: 90px 25px 8px 25px;
+  background-color: var(--purple);
+  border-top-left-radius: 70px;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(../img/footer-background.svg);
+  background-repeat: no-repeat;
+  background-position: top 40px right;
+}
+
+.contant-form__header {
+  font-size: 40px;
+  line-height: 52px;
+  color: var(--white);
+}
+
+.contact_form__paragraph {
+  font-size: 20px;
+  line-height: 28px;
+  font-weight: 400;
+  color: var(--tag-background);
+  margin-bottom: 50px;
+}
+
+.contact-form__black-bar {
+  display: flex;
+  justify-content: center;
+  margin-top: 32px;
+}
+
+@media screen and (min-width: 800px) {
+  body {
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  input[type="text"],
+  input[type="email"],
+  textarea {
+    border-radius: 0;
+  }
+
+  header {
+    padding: 30px 155px;
+  }
+
+  .header-menu-button {
+    width: auto;
+    height: 40px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-end;
+    padding: 0;
+    cursor: pointer;
+    user-select: none;
+  }
+
+  .header-menu-button > div {
+    width: 100px;
+    height: 20px;
+    margin-left: 10px;
+    background-color: transparent;
+    font-size: 15px;
+    color: #344563;
+  }
+
+  .header-menu-button:hover div {
+    background-color: transparent;
+    color: var(--text);
+  }
+
+  .headline {
+    background-image: url(../img/desktop-header-background.svg);
+    border-bottom-left-radius: 120px;
+  }
+
+  .generic-container {
+    background-color: white;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 142px 260px;
+    gap: 12px;
+  }
+
+  .headline.generic-container {
+    padding: 250px;
+  }
+
+  .generic_title {
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  .work-card {
+    height: auto;
+    background-color: var(--white);
+    border: 1px solid var(--border);
+    border-radius: 16px;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 40px;
+    gap: 50px;
+    margin: 0 10%;
+    width: calc(100% - 20%);
+  }
+
+  .work-card:nth-child(even) {
+    flex-direction: row-reverse;
+  }
+
+  .work-card:hover {
+    border: 2px solid #a7aeff;
+    cursor: pointer;
+    filter: drop-shadow(0 48px 48px rgba(37, 47, 137, 0.08));
+    scale: 1.03;
+  }
+
+  .work-card_elements {
+    width: 50%;
+  }
+
+  .work-card_picture {
+    width: 50%;
+  }
+
+  .work-card_description {
+    font-size: 16px;
+  }
+
+  .about-myself {
+    border-top-right-radius: 120px;
+    flex-direction: row;
+    gap: 40px;
+  }
```

Comment:
- [optional] I'm so sorry that to highlight this, please kindly ensure to align all the elements in this container according to the given template. This will boost your confidence the more when interacting with professional. 😄 Working perfectly with template shows that we are capable to do whatsoever is expected of us. 👍